dev-data/slider-photos.data.ts

```diff
@@ -0,0 +1,57 @@
+import photo1 from '@/shared/assets/photo-1.webp';
+import photo2 from '@/shared/assets/photo-2.webp';
+import photo3 from '@/shared/assets/photo-3.webp';
+import photo4 from '@/shared/assets/photo-4.webp';
+import photo5 from '@/shared/assets/photo-5.webp';
+import photo6 from '@/shared/assets/photo-6.webp';
+import photo7 from '@/shared/assets/photo-7.webp';
+import photo8 from '@/shared/assets/photo-8.webp';
+import photo9 from '@/shared/assets/photo-9.webp';
+
+export const sliderPhotos = [
+  {
+    id: 1,
+    src: photo1,
+    alt: 'Large group of conference attendees indoors, smiling and waving at the camera, with lanyards and name tags',
+  },
```

Comment:
a bad idea to put text this long to an alt attribute

https://www.w3.org/WAI/GL/WCAG20/tests/test3.html